app/lib/actions/pulp/consumer/sync_capsule.rb

```diff
@@ -14,6 +14,10 @@ def humanized_name
         def invoke_external_task
           pulp_resources.repository.sync(input[:repo_pulp_id])
         end
+
+        def run_progress_weight
+          100
```

Comment:
@zjhuntin this is for the progress bar, this task will be weighted heavily in the status of the progress, meaning the tasks before this one will be weighted less in proportion (depending on their weights).